recipes/links/build.sh

```diff
@@ -1,16 +1,7 @@
 #!/bin/sh
 set -eux -o pipefail
 
-./configure && make
-
-mkdir -p ${PREFIX}/bin/
-mkdir -p ${PREFIX}/bin/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/bin
-mkdir -p ${PREFIX}/bin/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/src
-
-cp src/LINKS_CPP ${PREFIX}/bin/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/src
-cp bin/* ${PREFIX}/bin/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/bin
-cp bin/LINKS* ${PREFIX}/bin/
-
+./configure && make && make install
```

Comment:
Can you change this line to 
`./configure --prefix=${PREFIX}`
 `make`
 `make install`